courses/migrations/0020_courseruncertificate_certificate_page_revision.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 3.2.15 on 2022-08-29 08:44
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def set_current_certificate_revision(apps, schema_editor):
```

Comment:
Do we need [this change](https://github.com/mitodl/mitxonline/pull/903/files#diff-64ebda3d82e0c746d2c838699c07acb411fbd29f3b343cf80597b3d1aa24740cR172-R173) after the data migration?

Reply:
Yes, This is for an edge case. If the `certificate page` is not yet created at any point for a course at that point if the certificate generated will have no revision. So this will add the Revision whenever the page is available and the certificate is accessed.